### ISSUE TITLE
Follow up content issues with Accessibility Working Group (AWG)

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual#1163]
Here's a list of issues that we'd like to run by the AWG at their next meeting. 

Some of them are for info, some for approval, but there is a wider issue of how the service manual team engages the AWG in updating the accessibility guidance. It might be worth dealing with that issue at the end, as the earlier issues are examples of the kinds of changes we need their input to.

**Comment by sarawilcox** [comment]
### Changes to the accessibility guidance

#### 1. Alt-text - proposal to keep to 125 characters or less

A recommendation from the mission team working on skin symptoms. 

In the accessibility guidance, in the section about informative images, we currently say:  

> "Keep alt-text to a sentence or 2 and no longer than 250 characters." 

At the March Content Style Council meeting, on the advice of the mission team working on skin symptoms, we agreed to change it to: 

> "Keep alt-text to a sentence or 2 and ideally under 125 characters including spaces.

This has been checked with Alistair Duggin. Some assistive tech will truncate at 125 characters while recent screen readers may extend to 150 characters. Occasionally it might be important to include information beyond this limit, even if it truncates for some users. More information in the [GitHub issue for alt-text: propose to keep 125 characters or less](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/308). 

After the March Style Council meeting, I posted about this in the AWG Slack channel, wondering if we could get it approved there: https://nhsuk.slack.com/archives/CNR01HHHP/p1618222658010100.

**Comment by sarawilcox** [comment]
#### 2. Update multimedia section of accessibility guidance

Current guidance: 
![Screenshot 2021-04-23 at 10 55 06](https://user-images.githubusercontent.com/45792959/115854703-6aa56680-a422-11eb-8c1b-70ab43cd0be8.png)
 
New guidance, updated with input from Rich Kelly who has consulted Michael Cheung, Leigh and Alistair Duggin. 
![Screenshot 2021-04-23 at 10 55 06](https://user-images.githubusercontent.com/45792959/115860206-1487f180-a429-11eb-9554-b4e2b8c02a3b.png)
![Screenshot 2021-04-20 at 10 53 20](https://user-images.githubusercontent.com/45792959/115860257-2a95b200-a429-11eb-966b-5fc9fd6ffdb7.png)

Ellen has run this content by the NHSD deaf awareness group. We're waiting for feedback. We may publish this content before the AWG but can iterate it if the AWG has any concerns/feedback. 


 (Question: 
There is a question from me, if something has been approved by Alistair and/or other members of the AWG, does it need approving by a full AWG meeting? Or can they just approve it on Slack?


we have issues in GitHub for every page in the guidance. They've been assigned to Michael Cheung.
What's happening about the issues Adrian raised about accessibility guidance pages, and the review Alistair Duggin was going to carry out?

**Comment by sarawilcox** [comment]
#### 3. PDF guidance

We updated the guidance in the content style guide after the January Content Style Council meeting: https://nhsuk-service-manual-dev-uks.azurewebsites.net/content/pdfs-and-other-non-html-documents

Changes to the content style guide don't need AWG approval but it is something the AWG might like to comment on and be aware of.

I have asked Amelia England or Mark Robinson (both content designers on the AWG) to show it to the AWG at the next meeting.

**Comment by sarawilcox** [comment]
### The wider issue of how the service manual team engages with the AWG in updating the accessibility guidance

In principle, I think we're keen that the AWG owns the accessibility guidance and oversees its development and any changes. But we haven't yet agreed how that will work in practice.

We now invite people to comment on pages in the guidance. Like other pages in the service manual, all the pages in the accessibility guidance have a call to action at the bottom of the page, inviting people to contribute. 

![Screenshot 2021-04-23 at 11 54 26](https://user-images.githubusercontent.com/45792959/115861507-c378fd00-a42a-11eb-8ac4-8c862d68ad20.png)

Every page in the guidance has an issue in GitHub and they're assigned to Michael Cheung, who will get notifications of any comments.

#### Questions:

- Will the AWG be testing the guidance and recommending changes? I think Alistair may have identified some changes we need to make.
- If people comment on the pages, how will the AWG review them and make any proposals for change?
- When people identify changes (like the changes mission team 1 and Rich identified above), what's the process of getting them approved by AWG? 
- If something is approved by Alistair or Leigh or Michael, does it need approval by a full AWG meeting? Or can we get things approved on Slack in the meantime to avoid hold-ups?

**Comment by sarawilcox** [comment]
### Upcoming change 

#### Setting page titles - section in accessibility guidance

Current guidance:

![Screenshot 2021-04-23 at 12 20 05](https://user-images.githubusercontent.com/45792959/115864101-549da300-a42e-11eb-97e6-3d82460710a4.png)

We have identified some confusion around this section. I'm checking in with content designers and devs and will redraft this section. What's the best way to get it signed off quickly?

**Comment by sarawilcox** [comment]
Hi @EllenDoyle and @georginaplatt, is this OK? I'll note any further changes on this ticket. Please feel free to come back to me if you have any queries.

**Comment by georginaplatt** [comment]
Yes great thankyou @sarawilcox 
I spoke to Leigh Friday the AWG is having a reset.
In the mean time he suggested to get a slot with Alistair Duggan to go through the list, not this Wed, next. I will email Sarah Norman to get a slot.

**Comment by sarawilcox** [comment]
@georginaplatt , I wonder if we could keep this ticket open till we get a chance to address them at the meeting. Or is it better if we create another ticket for that?

**Comment by georginaplatt** [comment]
1. Alt-text - 125 characters or less. 99% this works ok. 
2. Audio description not required if video supplements text already in the page (& video is abled as such). 
3. PDFs - options are to replace with HTML or delete. Alistair said another option is to make sure PDF is WCAG compliant & he has a PDF checklist. However is hard to check without professional audit. 
Not technically correct to say you cannot change colours PDFs. Is checking Adobe guidelines. Will add to teams chat.
